onmt/translate/translator.py

```diff
@@ -496,6 +499,10 @@ def _translate(
                 n_best_preds = [
                     " ".join(pred) for pred in trans.pred_sents[: self.n_best]
                 ]
+
+                n_best_scores = list(map(lambda x: x.item(),
+                                         trans.pred_scores[: self.n_best]))
```

Comment:
A list comprehension might be a bit more readable (and coherent with the operations above).
https://stackoverflow.com/questions/1247486/list-comprehension-vs-map

Reply:
done